encrypted-media.js

```diff
@@ -159,6 +159,9 @@
     'session-id': { func: term_helper, fragment: 'session-id', link_text: 'Session ID'  },
     'distinctive-identifier': { func: term_helper, fragment: 'distinctive-identifier', link_text: 'Distinctive Identifier'  },
     'distinctive-identifiers': { func: term_helper, fragment: 'distinctive-identifier', link_text: 'Distinctive Identifiers'  },
+    'record-of-key-usage': { func: term_helper, fragment: 'record-of-key-usage', link_text: 'record of key usage'  },
+    'first-decryption-time': { func: term_helper, fragment: 'first-decryption-time', link_text: 'first decryption time'  },
```

Comment:
I'm not sure I understand: it was your earlier comment which caused me to use <dfn> for these two and that seemed to necessitate adding these definitions to the JS in order to get links to work.